src/openshift/component.ts

```diff
@@ -261,13 +261,8 @@ export class Component extends OpenShiftItem {
 
         if (!repoURI) return null;
 
-        const gitRef = await vscode.window.showInputBox({
-            value: 'master',
-            prompt: 'Specify git reference (branch/tags/commits | default: master)',
-            validateInput: (value: string) => {
-                if (!value.trim()) return 'Empty reference, specify master if no reference needed.';
-            }
-        });
+        const references = await Refs.fetchTag(repoURI);
+        const gitRef = await vscode.window.showQuickPick([ ...references.keys()], {placeHolder: "Specify git reference (branch/tags)"});
```

Comment:
Should be `Select git reference (branch/tag)`.

Reply:
Fixed.